nltk/tokenize/stanford_segmenter.py

```diff
@@ -76,7 +75,7 @@ def __init__(self,
 
         # This is passed to java as the -cp option, the old version of segmenter needs slf4j.
         # The new version of stanford-segmenter-2016-10-31 doesn't need slf4j
-        # When using old version, set 'path_to_slf4j=YOUR_PATH' in the initilization of StanfordSegmenter
+        # When using old version, set path_to_slf4j='/YOUR_PATH/slf4j-api.jar' in the initilization of StanfordSegmenter
```

Comment:
I think you have decided that we would choose the plan A, which is letting user input the full path of slf4j-api.jar. Hence I change the parameter back to path_to_slf4j.